cluster/hooks.go

```diff
@@ -149,7 +150,28 @@ func InstallLogging(input interface{}, param pkgCluster.PostHookParam) error {
 	if err != nil {
 		return err
 	}
-
+	if loggingParam.TLSEnabled {
+		req := &secret.CreateSecretRequest{
+			Name: "tls-for-logging-operator",
+			Type: secretTypes.TLSSecretType,
+			Tags: []string{"logging-operator"},
+			Values: map[string]string{
+				secretTypes.TLSHosts: "fluent",
+			},
+		}
+		_, err := secret.Store.Store(cluster.GetOrganizationId(), req)
+		if err != nil {
+			return errors.Errorf("Failed generate TLS secrets to logging operator")
+		}
+		_, err = InstallOrUpdateSecrets(cluster,
+			&pkgSecret.ListSecretsQuery{
+				Type: secretTypes.TLSSecretType,
+				Tag:  "logging-operator",
```

Comment:
`logging-operator` should be passed in via a variable or const to avoid code duplication.